### PR TITLE
Feature/monitoring certificate

### PR DESCRIPTION
**Pull request by Talits**

### CODE REVIEW

File: ansible/playbooks/getup-certificate-monitoring/config.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- name: Check cert expirys
```
Comment (caruccio) [inline ansible/playbooks/getup-certificate-monitoring/config.yml]: expires

File: ansible/roles/getup-certificate-monitoring/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+- name: Check cert expirys on host
```
Comment (caruccio) [inline ansible/roles/getup-certificate-monitoring/tasks/main.yml]: expires

File: bin/certificate-monitoring [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+for i in ~/getup-engine-installer/state/*; do
+
+  cert="$i/certs"
+  NAME=$(grep CLUSTER_ZONE $i/terraform-out.env  | head -n1 | cut -d '=' -f2 | sed -e 's/\"//g')
```
Comment (caruccio) [inline bin/certificate-monitoring]: Must check for APPS_ZONE too.

Comment (Talits) [reply bin/certificate-monitoring]: It's only string , not a resource

**Pull request closed without merging**